lib/decidim/decidim_awesome/awesome.rb

```diff
@@ -43,6 +43,10 @@ module DecidimAwesome
       false
     end
 
+    config_accessor :allow_to_edit_proposals_after_import do
+      true
```

Comment:
```suggestion
      false
```
Let's default to the original value here